sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -210,6 +264,29 @@ void setRegionsContacted(Set<URI> regionsContacted) {
         this.regionsContacted = regionsContacted;
     }
 
+    private void printSystemInformation(StringBuilder stringBuilder) {
+        try {
+            long totalMemory = Runtime.getRuntime().totalMemory() / 1024;
+            long freeMemory = Runtime.getRuntime().freeMemory() / 1024;
+            long maxMemory = Runtime.getRuntime().maxMemory() / 1024;
+            String used_Memory = totalMemory - freeMemory + " KB";
+            String available_Memory = (maxMemory - (totalMemory - freeMemory)) + " KB";
+
+            OperatingSystemMXBean mbean = (com.sun.management.OperatingSystemMXBean)
```

Comment:
OperatingSystemMXBean from java.lang.management does not have precise information of CPU usage in percentage , therefore using com.sun.management. 
Java.lang have https://docs.oracle.com/javase/7/docs/api/java/lang/management/OperatingSystemMXBean.html#getSystemLoadAverage(), which after running on VMs unable to get the the correlation with usage properly

Reply:
This is oracle specific implementation. There are other JVMs too: IBM, Azul, Openjdk, etc, if the JVM is not oracle based implementation is com.sun.management  available?

Reply:
Openjdk has  com.sun.management available , and is giving numbers. I will try with another jdk maybe IBM

Reply:
Tested on AdoptSDk,IBM SDK , ZULU  they all have support for com.sun.management